includes/functions/admin/conditionals.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*******************************************************************************
- * Copyright (c) 2019, Code Atlantic LLC
+ * Copyright (c) 2023, Code Atlantic LLC
  ******************************************************************************/
```

Comment:
@s-acv2 - Looks like you missed this one for full header with description & package